izpack-installer/src/main/java/com/izforge/izpack/installer/console/AbstractTextConsolePanel.java

```diff
@@ -194,4 +191,11 @@ protected String removeHTML(String text)
         }
         return result;
     }
+
+    private String remoteHTMLTag(String text, String tag)
```

Comment:
I guess you meant `removeHTMLTag` as a method name here?

Reply:
Yes, corrected the typo.